### PR TITLE
Tweak 'ShortcutsView' style to make it looks more polished

### PR DESCRIPTION
**Pull request by otapliger**
screenshot

### CODE REVIEW

File: data/application.css [hunk 1]
```diff
@@ -14,3 +14,15 @@ decoration:backdrop {
         0 3px 6px alpha (#000, 0.23),
         0 14px 28px transparent;
 }
+
+.shortcuts-view {
```
Comment (cassidyjames) [inline data/application.css]: These should probably be done using the built-in Gtk margin properties instead of CSS.

Comment (otapliger) [reply data/application.css]: I see. Will work on that. Is there any technical reason or is just a code design choice?

**Comment by cassidyjames** [comment]
@danrabbit thoughts on the increased whitespace here?

**Comment by danrabbit** [comment]
Closing in favor of #62

**Pull request closed without merging**